src/Aspire.Dashboard/Components/ResourcesGridColumns/LogMessageColumnDisplay.razor

```diff
@@ -8,47 +8,59 @@
-                <FluentButton
-                    Id="@copyButtonId"
-                    Appearance="Appearance.Lightweight"
-                    OnClick="@(() => CopyTextToClipboardAsync(errorInfo, copyButtonId))">
-
-                    @ControlsStringsLoc[nameof(ControlsStrings.GridValueCopyToClipboard)]
-
-                    <FluentIcon Class="copy-icon align-text-bottom" Style="display:inline;" Icon="Icons.Regular.Size16.Copy"/>
-                    <FluentIcon Class="checkmark-icon align-text-bottom" Style="display:none;" Icon="Icons.Regular.Size16.Checkmark"/>
-                </FluentButton>
+<FluentStack HorizontalGap="0" Orientation="Orientation.Horizontal" Style="align-items: center;">
+    @if (TryGetErrorInformation(out var errorInfo))
+    {
+        var iconId = Guid.NewGuid().ToString();
+
+        @*
+            We don't want a browser tooltip to display when hovering over the error icon.
+            Use alt instead of title for screen readers. Also, the column has a title set
+            so add a whitespace title to prevent a browser tooltip from displaying.
+        *@
+        <FluentIcon alt="@Loc[nameof(Columns.LogMessageColumnExceptionDetailsTitle)]"
+                    title="@Loc[nameof(Columns.LogMessageColumnExceptionDetailsTitle)].ToString()"
+                    Style="flex-shrink: 0"
+                    Id="@iconId"
+                    Icon="Icons.Filled.Size16.DocumentError"
+                    Color="Color.Accent"
+                    Class="severity-icon"/>
+
+        var copyButtonId = Guid.NewGuid().ToString();
+
+        <FluentTooltip Anchor="@iconId" MaxWidth="650px">
+            <div style="padding: 10px">
+                <h4>@Loc[nameof(Columns.LogMessageColumnExceptionDetailsTitle)]</h4>
+
+                <pre style="white-space: pre-wrap; overflow-wrap: break-word; max-height: 300px; overflow: auto;">@errorInfo</pre>
+
+                <div style="float: right; margin-bottom: 10px;">
+                    <FluentButton
+                        Id="@copyButtonId"
+                        Appearance="Appearance.Lightweight"
+                        OnClick="@(() => CopyTextToClipboardAsync(errorInfo, copyButtonId))">
+
+                        @ControlsStringsLoc[nameof(ControlsStrings.GridValueCopyToClipboard)]
+
+                        <FluentIcon Class="copy-icon align-text-bottom" Style="display:inline;" Icon="Icons.Regular.Size16.Copy"/>
+                        <FluentIcon Class="checkmark-icon align-text-bottom" Style="display:none;" Icon="Icons.Regular.Size16.Checkmark"/>
+                    </FluentButton>
+                </div>
             </div>
-        </div>
-    </FluentTooltip>
-}
+        </FluentTooltip>
 
-<FluentHighlighter HighlightedText="@FilterText" Text="@LogEntry.Message" />
+        <GridValue Value="@LogEntry.Message"
+                   EnableHighlighting="true"
+                   HighlightText="@FilterText"
+                   BackgroundColor="transparent" />
```

Comment:
Applying BackgroundColor to a FluentButton (Which is what this has the effect of doing) puts the FluentButton into sort of an odd mode where the styling is detached from the normal fluent styling. In fact, I just noticed there's a bug where it generates invalid inline style blocks too (I'll file that shortly with the library).

I think there's an easier way to fix the complaint that James had about losing the hover color:

1. Back out the changes in the most recent commit (https://github.com/dotnet/aspire/pull/2344/commits/4f254e4c60932df644b6af8f0a907a2726a05b82)
2. Remove the two `BackgroundColor="transparent"` attributes from GridValue.razor that were added in https://github.com/dotnet/aspire/pull/2344/commits/7c065ed4b709f3c1349c2a27e00d196be288ca87
3. In StructuredLogs, change these lines:

```css
::deep .log-row-critical {
    background-color: var(--log-critical);
}

::deep .log-row-error {
    background-color: var(--log-error);
}

::deep .log-row-warning {
    background-color: var(--log-warning);
}
```

to this:

```css
::deep .log-row-critical,
::deep .log-row-critical fluent-button[appearance=lightweight]:not(:hover)::part(control) {
    background-color: var(--log-critical);
}

::deep .log-row-error,
::deep .log-row-error fluent-button[appearance=lightweight]:not(:hover)::part(control) {
    background-color: var(--log-error);
}

::deep .log-row-warning,
::deep .log-row-warning fluent-button[appearance=lightweight]:not(:hover)::part(control){
    background-color: var(--log-warning);
}
```

That has the side effect of making the View button in the last column also "transparent":

![image](https://github.com/dotnet/aspire/assets/9613109/c024f9e6-77da-4cfe-a3d4-d7a472d0b3eb)

Hover colors on the copy button and the View button are then consistent as well (they were different in the current commit of this PR because of the opacity mode that the FluentButton got into).

Reply:
> (I'll file that shortly with the library).

Filed https://github.com/microsoft/fluentui-blazor/issues/1598